### DISCUSSION TITLE
Contribute to project

### DISCUSSION THREAD

**Discussion by sumanjha108155** [nirajgiriXD/garden-of-quotes#15] [Q&A] [answered]
How can we contribute to this project ?

**Comment by nirajgiriXD** [comment, marked as answer]
In order to contribute to the project, you can follow these steps:
- Fork the repository
- Make changes to the code or simply add a new quote
- Create a PR

You may follow the [WORKFLOW](https://github.com/nirajgiriXD/garden-of-quotes/blob/main/WORKFLOW.md) file for better clarity.
